docker/zookeeper/versions.yaml

```diff
@@ -25,8 +25,8 @@ versions:
       gpg: 52A7EA3EECAE05B0A8306471790761798F6E35FC
       version: 3.8.2
     gosu:
-      version: 1.16
       gpg: B42F6819007F00F88E364FD4036A9C25BF357DD4
+      version: '1.16'
```

Comment:
Looking at the commit message as update the version but seeing the version but with extra '. Wanted to double check make sure we're submitting the right change

Reply:
Actually, commit message and contents are automatically generated by an internal updater tool. In this case, gosu has been added to a current bugfix and then, by code-generated, order has been changed. 👍🏻 

Actually the change is just enclosing the `1.16` into single-quotes.